Source/Core/Core/ConfigLoaders/GameConfigLoader.cpp

```diff
@@ -272,7 +275,7 @@ void INIGameConfigLayerLoader::Save(Config::Layer* layer)
     const Config::Location& location = config.first;
     const std::optional<std::string>& value = config.second;
 
-    if (!IsSettingSaveable(location))
+    if (!IsSettingSaveable(location) || location.system == Config::System::Session)
```

Comment:
Shouldn't we just modify `IsSettingSaveable` to return false for `location.system == Config::System::Session` instead?

Reply:
`IsSettingSaveable` marks settings that are still handled by the old config system, and based on code comments, it seems like we intend to drop `IsSettingSaveable` when the old config system is dead. The `Session` system is already handled by the new config system, and we want to keep not saving in the future when the old config system is gone.

Reply:
Ah, yeah okay. I guess `Session` is where anything that we don't want to be persistent should go. I'm gonna make a follow-up PR that moves a few of such settings there.

Reply:
It still feels a bit weird that `Config::System::Session` shows up scattered around everywhere. Isn't this what the layered configuration system was designed to solve in first place? Have a layer that is session-only, doesn't save, and has that single setting?

Reply:
The layered config system supports session-only *values*, but not session-only *settings*. I want to make it impossible for this setting to get loaded from Dolphin.ini so that nobody overrides the default by accident.